### ISSUE TITLE
[ZE] Alterar beneficios de cla tag para a tag de grupo da steam.

### ISSUE THREAD

**Issue by InFuse5** [quasemago/zrageservers#177]
Antes o motivo para não fazer essa alteração era de que conflitaria com a entwatch, entretanto as cla tags nem estão sendo usadas mais para ewatch, então acho uma boa alterar os beneficios para a tag de grupo da steam, ja que é melhor que aquele negocio feio no nome @zrage eu acho.

**Comment by lpcorrea** [comment]
A tag do grupo hoje é `ZM RAGE |` não seria interessante mudar para `ZRAGE |`?

![download](https://user-images.githubusercontent.com/50057540/94947779-9c6dd480-04b4-11eb-8560-a192f518ab2a.png)

**Comment by gabrielacms** [comment]
Coloca uma skin bem leve (lara croft) para o pessoal que usar o @Zrage 

- Gosto do ZRAGE ao invés do ZM RAGE.

**Comment by alminha021** [comment]
Zm Rage é horroroso. Concordo em trocar a abreviação.

**Comment by quasemago** [comment]
Clantag é reservada e "ZRAGE" já está em uso, por esse motivo alguns servidores de Zombie Escape colocam caracteres especiais.

**Comment by alminha021** [comment]
Nao vejo problema em colocar o " | " , pra ficar " ZRAGE | "
"ZRAGE | alm a "

por mim de boa, trocando esse zm rage que eh horrivel ja ta bom.

**Comment by danielhidalgojunior** [comment]
prefiro zRage na tag, em minúsculo. O "|" pra mim tanto faz

**Comment by InFuse5** [comment]
Coloca um caractere a mais alem da zrage pra fica bonito tipo ZRAGE |  como falaram ali ou ZRAGE™

**Comment by quasemago** [comment]
Feito.

**Comment by quasemago** [comment]
O que acham da ideia de adicionar algo para o discord? Exemplo, o jogador teria que ir no discord e vincular seu steamid com o bot como "!vincular meusteam" (provavelmente seria por link do perfil pra facilitar pros batata que não sabe o que é steamid).

Claro, teria que ficar de olho pra ninguém registrar a steamid de outra pessoa.

**Comment by danielhidalgojunior** [comment]
só de o cara vincular a steam com o discord já n funciona?

**Comment by quasemago** [comment]
@danielhidalgojunior não, api do discord não da expose nisso para os bots, unica forma de obter isso é fazer gambiarra criando um usuário normal que funcione como bot (mas isso é proibido e qualquer report da ban na conta).

**Comment by danielhidalgojunior** [comment]
se for fazer isso, tem q ter um cuidado com a questão da segurança. Poderia ter um 2step de confirmação, o cara usa o comando no ds, recebe um código e usa ele com algum cmd dentro do jogo.

**Comment by lpcorrea** [comment]
No servidor argentino ele primeiro pega pelo CS, manda um !verify no chat, dai ele recebe um token e coloca no discord "!verify token"

**Comment by InFuse5** [comment]
E qual seria o beneficio ? pq batata ja se embanana com coisa simples, imagina com coisa burocratica demais.

**Comment by quasemago** [comment]
> se for fazer isso, tem q ter um cuidado com a questão da segurança. Poderia ter um 2step de confirmação, o cara usa o comando no ds, recebe um código e usa ele com algum cmd dentro do jogo.
@danielhidalgojunior Sim, da pra fazer. Também poderia ser ao contrário, o cara usa um comando no game e gera um id único para ele, então no discord ele confirma.

**Comment by quasemago** [comment]
> E qual seria o beneficio ? pq batata ja se embanana com coisa simples, imagina com coisa burocratica de mais.

Bom, como seria um beneficio não tem problema batata não saber fazer isso, problema seria dele.

**Comment by quasemago** [comment]
Creio que esse recurso ficou meio largado agora, visto que alguns benefícios foram transferidos para a verificação via discord. Tendo alguns necessitando estar no grupo da steam e verificado (ver através do comando **!bonus**).

Adicionar outra dependência pro jogador acessar tais recursos não acho uma boa ideia.

**Comment by InFuse5** [comment]
> Creio que esse recurso ficou meio largado agora, visto que alguns benefícios foram transferidos para a verificação via discord. Tendo alguns necessitando estar no grupo da steam e verificado (ver através do comando **!bonus**).
> 
> Adicionar outra dependência pro jogador acessar tais recursos não acho uma boa ideia.

Mas isso é so pra credito em outros servers, no nosso caso credito e hcredito em dobro.

**Comment by quasemago** [comment]
> Mas isso é so pra credito em outros servers, no nosso caso credito e hcredito em dobro.

Hmm, irei deixar isso para ser implementado antes do próximo holiday.
